docs/configuration/istio/ingress.md

```diff
@@ -0,0 +1,83 @@
+
+### Enable Passthrough Gateway
+
+In order to enable the Passthrough Gateway, you must specify `istio-passthrough-gateway` as an `optionalComponent` in your UDS Bundle configuration. Here is an example of how to do this:
+
+```yaml
+kind: UDSBundle
+metadata:
+  name: core-with-passthrough
+  description: A UDS example bundle for packaging UDS core with the passthrough gateway enabled
+  version: "0.0.1"
+
+packages:
+  - name: core
+    repository: oci://ghcr.io/defenseunicorns/packages/uds/core
+    ref: 0.23.0-upstream
+    # You must specify the istio-passthrough-gateway as an optionalComponent or else it will not be deployed
+    optionalComponents:
+      - istio-passthrough-gateway
+```
+
+### Configure Domain Name and TLS for Istio Gateways
+
+By default, the UDS Core Istio Gateways are set up to use the `uds.dev` domain and have a valid TLS certificate packaged.  You will want to change the domain name for your environment and provide a valid TLS certificate for this domain.
+
+You can set the TLS certs via overrides in a [UDS Bundle](https://uds.defenseunicorns.com/bundles/) (see below).
+```yaml
+kind: UDSBundle
+metadata:
+  name: core-with-cert-override
+  description: A UDS example bundle for packaging UDS core with a custom TLS certificate
+  version: "0.0.1"
+  
+packages:
+  - name: core
+    repository: oci://ghcr.io/defenseunicorns/packages/uds/core
+    ref: 0.23.0-upstream
+    overrides:
+      istio-admin-gateway:
+        uds-istio-config:
+          variables:
+            - name: ADMIN_TLS_CERT
+              description: "The TLS cert for the admin gateway (must be base64 encoded)"
+              path: tls.cert
+            - name: ADMIN_TLS_KEY
+              description: "The TLS key for the admin gateway (must be base64 encoded)"
+              path: tls.key
+      istio-tenant-gateway:
+        uds-istio-config:
+          variables:
+            - name: TENANT_TLS_CERT
+              description: "The TLS cert for the tenant gateway (must be base64 encoded)"
+              path: tls.cert
+            - name: TENANT_TLS_KEY
+              description: "The TLS key for the tenant gateway (must be base64 encoded)"
+              path: tls.key
+```
+The environment variables `ADMIN_TLS_CERT`, `ADMIN_TLS_KEY`, `TENANT_TLS_CERT`, and `TENANT_TLS_KEY` should be base64 encoded strings of the TLS certificate and key for the admin and tenant gateways respectively.
+
+
+You can then set the domain via [uds-config](https://uds.defenseunicorns.com/cli/quickstart-and-usage/#variables-and-configuration) file using shared key to override the Zarf Domain Variable (see below `uds-config.yaml`).
+```yaml
+shared:
+   domain: yourawesomedomain.com # shared across all packages in a bundle
+```
```

Comment:
Might be useful to provide the TLS cert/keys as config file examples as well? If specified as environment variables they would need to be prefixed with `UDS_` for uds cli to actually pick them up.

Reply:
```suggestion

You can then either use environment variables (`UDS_ADMIN_TLS_CERT`, `UDS_ADMIN_TLS_KEY`, `UDS_TENANT_TLS_CERT`, and `UDS_TENANT_TLS_KEY`) or a config file to set the values for each gateway. These values should be base64 encoded strings of the TLS certificate and key for the admin and tenant gateways respectively.

Domain should be set via your [uds-config](https://uds.defenseunicorns.com/cli/quickstart-and-usage/#variables-and-configuration) file using the shared key to override the Zarf Domain Variable (see example `uds-config.yaml`).

```yaml
shared:
  domain: yourawesomedomain.com # shared across all packages in a bundle

# TLS Certs/Keys if not provided via environment variables
variables: 
  core:
    admin_tls_cert: # base64 encoded admin cert here
    admin_tls_key: # base64 encoded admin key here
    tenant_tls_cert: # base64 encoded tenant cert here
    tenant_tls_key: # base64 encoded tenant key here
```
```

Reply:
I think the suggestion trimmed off a trailing end of code block, but that's what I'd envision for the updated content.

Reply:
added suggestions! thanks for explaining this and I'm glad we will have easy snippets to use now (instead of trying to navigate all the overrides and naming conventions 😂 ).